packages/react-components/react-list-preview/src/components/List/useList.ts

```diff
@@ -9,23 +19,70 @@ import type { ListProps, ListState } from './List.types';
  * before being passed to renderList_unstable.
  *
  * @param props - props from this instance of List
- * @param ref - reference to root HTMLDivElement of List
+ * @param ref - reference to root HTMLElement of List
  */
-export const useList_unstable = (props: ListProps, ref: React.Ref<HTMLDivElement>): ListState => {
+export const useList_unstable = (props: ListProps, ref: React.Ref<HTMLDivElement | HTMLUListElement>): ListState => {
+  const {
+    layout = 'vertical',
+    focusableItems = false,
+    selectable = false,
+    selectionMode = 'multiselect',
+    selectedItems,
+    defaultSelectedItems,
+    as,
+    onSelectionChange,
+  } = props;
+
+  const arrowNavigationAttributes = useArrowNavigationGroup({
+    axis: layout === 'grid' ? 'grid-linear' : 'both',
+    memorizeCurrent: true,
+  });
+
+  const [selectionState, setSelectionState] = useControllableState({
+    state: selectedItems,
+    defaultState: defaultSelectedItems,
+    initialState: [],
+  });
+
+  const onChange = useEventCallback((e: React.SyntheticEvent<Element, Event>, data: OnSelectionChangeData) => {
+    const selectedItemsAsArray = Array.from(data.selectedItems);
+    setSelectionState(selectedItemsAsArray);
+    onSelectionChange?.(e, { selectedItems: selectedItemsAsArray });
+  });
+
+  const selection = useListSelection({
+    onSelectionChange: onChange,
+    selectionMode,
+    selectedItems: selectionState,
+    defaultSelectedItems,
+  });
+
+  const selectableListProps = React.useMemo(
+    () => ({
+      role: 'listbox',
+      'aria-multiselectable': selectionMode === 'multiselect' ? true : undefined,
+    }),
+    [selectionMode],
+  );
```

Comment:
As there is no point to memoize strings, I suggest to move this object directly there:

<img width="427" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/61a243db-88fa-4809-8f9d-7a7393a25f6c">

That's up to you. Anyway, please remove `React.useMemo()` call